src/net/sourceforge/kolmafia/persistence/HolidayDatabase.java

```diff
@@ -22,6 +22,7 @@
   private static int RONALD_PHASE = -1;
   private static int GRIMACE_PHASE = -1;
   private static int HAMBURGLAR_POSITION = -1;
+  private static final TimeZone ROLLOVER = TimeZone.getTimeZone("GMT-0330");
```

Comment:
Yes. Static initialization is done in order. Therefore, ROLLOVER has to be defined before getPhaseStep (which uses it) is called in a static block.